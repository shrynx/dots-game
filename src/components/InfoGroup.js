// @flow
import styled from 'styled-components';

const InfoGroup = styled.div`
  flex: 1;
  align-self: center;
  text-align: center;
`;

export default InfoGroup;
