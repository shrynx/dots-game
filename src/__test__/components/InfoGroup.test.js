import React from 'react';
import { render } from 'react-testing-library';
import InfoGroup from '../../components/InfoGroup';

it('InfoGroup component should render correctly', () => {
  const { container } = render(<InfoGroup />);
  expect(container).toMatchSnapshot();
});
