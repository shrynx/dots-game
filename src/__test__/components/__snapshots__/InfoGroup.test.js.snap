// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoGroup component should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa MTTgg"
  />
</div>
`;
